arches_component_lab/src/arches_component_lab/widgets/ResourceInstanceMultiselectWidget/components/ResourceInstanceMultiselectWidgetEditor.vue

```diff
@@ -55,6 +55,7 @@ const isLoading = ref(false);
 const resourceResultsPage = ref(0);
 const resourceResultsTotalCount = ref(0);
 const fetchError = ref<string | null>(null);
+const emptyFilterMessage = ref($gettext("Search returned no results"));
```

Comment:
nit: I'd just change this to `filterMessage` since it's being altered later in the file.